include/code_800558C.h

```diff
@@ -5,13 +5,22 @@
 #include "structs/axdata.h"
 #include "file_system.h"
 
+typedef struct Palette256
+{
+    RGB pal[256];
+} Palette256;
+
 // size: 0x20
-struct axMapSprite
+struct EfoFileData
 {
-    axmain axmain;
-    const u32 *tiles;
-    const RGB *pal;
-    s32 tileCount;
+    /* 0x0 */ ax_pose **poses; // TODO: Verify if these fields' names are actually valid
+    /* 0x4 */ ax_anim ***animations;
+    /* 0x8 */ s32 animCount;
+    /* 0xC */ void *spriteData; // ?
+    /* 0x10 */ Palette256 *unk10; // ?
+    /* 0x14 */const u32 *tiles;
```

Comment:
Missing a space after the slash on these

Reply:
Space added